tldr.yy

```diff
@@ -30,6 +30,8 @@ more_information  : GREATER_THAN MORE_INFORMATION ANGLE_BRACKETED_URL END_MORE_I
                     -> yy.addMoreInformation($ANGLE_BRACKETED_URL)
                   | GREATER_THAN MORE_INFORMATION END_MORE_INFORMATION
                     -> yy.error(@$, 'TLDR017') || yy.addDescription($MORE_INFORMATION + $END_MORE_INFORMATION.trim())
+                  | more_information GREATER_THAN MORE_INFORMATION ANGLE_BRACKETED_URL END_MORE_INFORMATION_URL
```

Comment:
Should `MORE_INFORMATION`, `ANGLE_BRACKETED_URL`, and `END_MORE_INFORMATION_URL` be present in the `%token` definitions at the top of the file in some form, or is this different to regular bison syntax?

Reply:
I'm not actually sure... they weren't included in the PR that added them originally. 🤷🏻 I know pretty much nothing about Bison syntax.